tests/admin_filters/tests.py

```diff
@@ -1038,6 +1039,22 @@ def test_filter_with_failing_queryset(self):
         with self.assertRaises(ZeroDivisionError):
             modeladmin.get_changelist_instance(request)
 
+    def test_filter_for_change_list_form(self):
+        modeladmin = BookAdmin(Book, site)
+        cases = ({'q': '\x00'}, 'Null characters are not allowed'), \
```

Comment:
Any reason not to just use a list or explicit tuple here? (The `\` continuation is a little strange.)

Reply:
I tried to make a tuple which contains tuple items but I've done wrong. I fixed it